perllib/FixMyStreet/App/Controller/Around.pm

```diff
@@ -258,13 +258,11 @@ sub check_and_stash_category : Private {
     )->all;
     my @categories = map { $_->category } @contacts;
     $c->stash->{filter_categories} = \@categories;
-
-
-    my $category = $c->get_param('filter_category');
     my %categories_mapped = map { $_ => 1 } @categories;
-    if ( defined $category && $categories_mapped{$category} ) {
-        $c->stash->{filter_category} = $category;
-    }
+
+    my $categories = [ $c->get_param_list('filter_category', 1) ];
+    my @valid_categories = grep { $_ && $categories_mapped{$_} } @$categories;
+    $c->stash->{filter_category} = \@valid_categories;
```

Comment:
It's worse than that; filter_categories already exists, and is the list of categories to *show*.
Changing the request query param I'd be fine with, if not for that. I did try and change them all but got part way through and it all went horribly wrong so I rolled that bit back.